projects/packages/stats/tests/php/test-tracking-pixel.php

```diff
@@ -100,10 +81,16 @@ public function test_get_footer_to_add_amp_request() {
 			'srv'  => 'example.org',
 		);
 		add_filter( 'jetpack_is_amp_request', '__return_true' );
-		$footer_to_add = Tracking_Pixel::get_footer_to_add( $data );
+
+		$method = new \ReflectionMethod( 'Automattic\Jetpack\Stats\Tracking_Pixel', 'get_amp_footer' );
```

Comment:
Just a pet peeve, I find `::class` seems less likely to be missed when updating things than the string version.
```suggestion
		$method = new \ReflectionMethod( Tracking_Pixel::class, 'get_amp_footer' );
```
There are other places in the PR where the same might be done.

Reply:
👍 Done in e972b8544e63832fdfb3903979b30beb86bf1db2